forge-gui/res/cardsfolder/upcoming/the_horus_heresy.txt

```diff
@@ -0,0 +1,13 @@
+Name:The Horus Heresy
+ManaCost:3 U B R
+Types:Enchantment Saga
+K:Saga:3:GainControl,DBDraw,DBRepeat
+SVar:GainControl:DB$ GainControl | ValidTgts$ Creature.nonLegendary+OppCtrl | LoseControl$ LeavesPlay,LoseControl | TgtPrompt$ Select target nonlegendary creature an opponent controls to gain control of. | TargetMin$ OneEach | TargetMax$ OneEach | TargetsWithDifferentControllers$ True | SpellDescription$ For each opponent, gain control of up to one target nonlegendary creature that player controls for as long as CARDNAME remains on the battlefield.
```

Comment:
Is that TargetMin correct?
Also LoseControl shouldn't be both

Reply:
I've found that with Blatant Thievery. I haven't tested multiplayer yet but it works 1v1

Reply:
But that card doesn't say "up to"